src/gwt/src/org/rstudio/studio/client/workbench/ui/PaneManager.java

```diff
@@ -576,7 +576,21 @@ public void onFocusRightSeparator()
    @Handler
    public void onFocusCenterSeparator()
    {
-      panel_.focusSplitter();
+      panel_.focusSplitter(right_);
+   }
+
+   @Handler
+   public void onFocusSourceColumnSeparator()
+   {
+      LogicalWindow window = getParentLogicalWindow(sourceColumnManager_.getActiveElement());
+      if (window == null ||
+          window.getNormal() == null ||
+          !window.getNormal().getName().contains(SourceColumnManager.COLUMN_PREFIX))
+      {
+         Debug.log("Attempted focusSourceColumnSeparator when source column not active.");
```

Comment:
Did you mean to leave this in?

Reply:
Yes, in theory we should never reach this point because the command is only enabled when the active element belongs to an active source pane. However, if we attempt to call `panel_.focusSplitter(window.getNormal())` when Normal is null then we can except out. I decided to return as well as log a debug statement since if this code gets reached something else is probably wrong. Is this something we wouldn't typically log?

Reply:
I think it’s fine, be good to know if this happens. Was just checking.

Reply:
We don't usually leave `Debug.log` statements in; this should be `Debug.logWarning`.

Reply:
Fixed this 👍